Bottom: b91f6f81de13c26679a9a9200b6915392a36e550
Top:    6d595304d69b66b20b2f2d51c1664d83dad1866d
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:58 -0500

Cleaner method to get ip address


---
diff --git a/docker/docker-exim4/docker-entrypoint.sh b/docker/docker-exim4/docker-entrypoint.sh
index c6290553..e6fb458d 100755
--- a/docker/docker-exim4/docker-entrypoint.sh
+++ b/docker/docker-exim4/docker-entrypoint.sh
@@ -28,9 +28,6 @@ case ${1} in
             [ -e /etc/mailname ] \
                 && rm -rf /etc/mailname
             echo "${HOSTNAME}" > /etc/mailname
-            # set local IP addr
-            LOCAL_IP_ADDR=$(grep "[ \t]*${HOSTNAME}[ \t]*" /etc/hosts|cut -f 1)
-            sed -i 's|dc_local_interfaces=.*|dc_local_interfaces='"'${LOCAL_IP_ADDR}'"'|' ${CONFIG_PATH}/update-exim4.conf.conf
             # copy certificate
             [ -e /etc/ssl/private/exim.crt ] \
                 && rm -f ${CONFIG_PATH}/exim.crt \
